### PR TITLE
fix test download of svg images

### PR DESCRIPTION
**Pull request by car-roll**
PromotedBuildActionTest fails for jenkins versions > 2.221 due to UI change.
see https://github.com/jenkinsci/jenkins-test-harness/pull/200

### Your checklist for this pull request

- [x] Pull request title represents the changelog entry which will be used in Release Notes. JIRA issue is a part of the title (see existing PRs as examples)
- [x] Please describe what you did. Short summary for reviewers. If the change involves WebUI, add screenshots 
- [ ] Link to relevant [Jenkins JIRA issues](https://issues.jenkins-ci.org) or pull requests
- [x] Test automation, if relevant. We expect autotests for all new features or complex bugfixes
- [ ] Documentation if relevant (new features). We want to use Documentation-as-Code, just put docs to README or to new Doc files
- [ ] Javadoc for new APIs. Any public/protected method is considered as API unless annotated by `Restricted` ([docs](https://kohsuke.org/access-modifier/apidocs/org/kohsuke/accmod/AccessRestriction.html)) 

### CC

@jglick

### CODE REVIEW

File: src/test/java/hudson/plugins/promoted_builds/PromotedBuildActionTest.java [hunk 1]
```diff
@@ -48,9 +49,11 @@ public void testDeletedPromotionProcess() throws Exception {
             } 
             HtmlImage img = (HtmlImage)candidate;
             try {
-                img.getHeight();
+                assertEquals("Failed to load " + img.getSrcAttribute(),
+                        200,
+                        img.getWebResponse(true).getStatusCode());
             } catch (IOException e) {
-                throw new IOException2("Failed to load "+img.getSrcAttribute(),e);
+                throw new AssertionError("Failed to load " + img.getSrcAttribute());
```
Comment (jglick) [inline src/test/java/hudson/plugins/promoted_builds/PromotedBuildActionTest.java:56]: Or just let it be thrown.

**Review by jglick — APPROVED** [review]
I guess? Do not know anything about this.

**Comment by jglick** [comment]
I am not a plugin maintainer, if that is why you were mentioning me?

**Comment by car-roll** [comment]
sorry, yes that is why I was. @oleg-nenashev would this be you? If so, I'm looking for a merge and release

**Comment by oleg-nenashev** [comment]
James Nord took ownership of this plugin a while ago.
He is on your team AFAIK :)

**Comment by jtnord** [comment]
> James Nord took ownership of this plugin a while ago.

I have no recolection of that, but ok then 😆

**Pull request merged**